src/user.js

```diff
@@ -46,7 +46,7 @@ class User {
       if(match === undefined) {
         return false
       }
-      recipeIng.quantity.amount <= match.amount
+      return recipeIng.quantity.amount <= match.amount
```

Comment:
I didn't catch the naming here. We can look at the classes and naming when we refactor.